homeassistant/components/mochad/light.py

```diff
@@ -49,74 +49,59 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up X10 dimmers over a mochad controller."""
-    mochad_controller = hass.data[DOMAIN]
-    devs = config[CONF_DEVICES]
+    mochad_controller: MochadCtrl = hass.data[DOMAIN]
+    devs: list[dict[str, Any]] = config[CONF_DEVICES]
     add_entities([MochadLight(hass, mochad_controller.ctrl, dev) for dev in devs])
 
 
 class MochadLight(LightEntity):
     """Representation of a X10 dimmer over Mochad."""
 
+    _attr_assumed_state = True  # X10 devices are normally 1-way
     _attr_color_mode = ColorMode.BRIGHTNESS
     _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
 
-    def __init__(self, hass, ctrl, dev):
+    brightness: int
```

Comment:
Brightness is a property method, its typing should not be overriden like this.

Reply:
I couldn't work out a better way so I used assert in the end